lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -868,6 +878,21 @@ def exec_command(self, command):
             stderr=subprocess.DEVNULL,
         )
 
+    def detect_init_system(self):
```

Comment:
```suggestion
    def detect_init_system(self) -> InitMode:
```
I think it's always good to document the return and input types of new functions.

Reply:
Good point, ta! I've added that now.